src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -181,16 +188,37 @@ export class QuickBar extends LitElement {
     }
 
     this._items = (this._commandMode ? this._commandItems : this._entityItems)
```

Comment:
We should do initial sorting when we create the `_commandItems` and  `_entityItems` so we can just return that when the filter is empty. So we only have to call `fuzzySequentialMatch` when we have a filter.

Reply:
The compare also be much simpler as we only compare numbers

Reply:
Ah, I didn't see this comment when I wrote my [other comment](https://github.com/home-assistant/frontend/pull/7367/files#r507866405). I agree. I'll make the change.

Reply:
Okay, that was _worlds_ easier. I'm not sure what I was thinking.